Guides and Surveys: Document new updateLanguage SDK method (#711)
amplitude/amplitude-docs: content/collections/guides_and_surveys/en/sdk.md

{{/partial:tab}}
{{/partial:tabs}}

> For additional configuration, you can supply `InitOptions` to the `plugin` function. See [below](#initialize-the-sdk) for the available options.

Reviewer: ```suggestion
For additional configuration, supply `InitOptions` to the `plugin` function. See [Initialize the SDK](#initialize-the-sdk) below for the available options.
```

    window.engagement.setRouter((newUrl) => navigate(newUrl));
  }, []);
};

## Localization

> Guides and Surveys can be localized by setting the `locale` option during initialization. This option is available in the `InitOptions` when using the [Amplitude Browser SDK](#amplitude-browser-sdk) plugin, and in the `options` for the `engagement.init()` method if you're using a [Third-party analytics provider](#third-party-analytics-provider).

Reviewer: ```suggestion
Set the `locale` option during initialization to localize a guide or survey.

* If you use the Amplitude Browser SDK](#amplitude-browser-sdk) plugin, set it in `InitOptions`.
* If you use a  [third-party analytics provider](#third-party-analytics-provider), set it in `options` within the `engagement.init()` method.
```

> To dynamically update the language after the SDK has been initialized, use the `updateLanguage` method documented below. Calling `updateLanguage` will re-fetch the configuration with the new locale.

Reviewer: ```suggestion
To dynamically update the language after the SDK initializes, use the `updateLanguage` method documented below. Calling `updateLanguage` re-fetches the configuration with the new locale.
```
